src/search/landmarks/exploration.cc

```diff
@@ -14,21 +14,17 @@
 using namespace std;
 
 namespace landmarks {
-/* Integration Note: this class is the same as (rich man's) FF heuristic
-   (taken from hector branch) except for the following:
-   - Added-on functionality for excluding certain operators from the relaxed
-     exploration (these operators are never applied, as necessary for landmark
-     computation)
-   - Exploration can be done either using h_max or h_add criterion (h_max is needed
-     during landmark generation, h_add later for heuristic computations)
-   - Unary operators are not simplified, because this may conflict with excluded
-     operators. (For an example, consider that unary operator o1 is thrown out
-     during simplify() because it is dominated by unary operator o2, but then o2
-     is excluded during an exploration ==> the shared effect of o1 and o2 is wrongly
-     never reached in the exploration.)
-   - Added-on functionality to check for a set of propositions which one is reached
-     first by the relaxed exploration, and extract preferred operators for this cheapest
-     proposition (needed for planning to nearest landmark).
+/*
+  Integration Note: this class is the same as (rich man's) FF heuristic
+  (taken from hector branch) except for the following:
+  - Added-on functionality for excluding certain operators from the relaxed
+    exploration. (These operators are never applied, as necessary for landmark
+    computation.)
+  - Unary operators are not simplified, because this may conflict with excluded
+    operators. (For an example, consider that unary operator o1 is thrown out
+    during simplify() because it is dominated by unary operator o2, but then o2
+    is excluded during an exploration ==> the shared effect of o1 and o2 is
+    wrongly never reached in the exploration.)
```

Comment:
I wonder if we should keep this comment at all; I don't know if anyone can still reproduce the FF heuristic from hector branch, and we changed a lot in the class in this revision.

Reply:
The "FF heuristic from hector branch" is our FF heuristic.

I think the first half of the comment is no longer useful and should be removed. I think the warning regarding "simplify" is still useful because it mentions a dangerous trap. I suggest rewriting the comment to only mention this, i.e., say that the simplification method in our delete relaxation (RelaxationHeuristic::simplify) cannot be used here for the reasons mentioned.